tensorflow/compiler/tf2xla/rearrange_function_argument.cc

```diff
@@ -146,7 +146,7 @@ Status ReorderOutputEdges(Graph* g, Node* n, int input_count,
     int dst_input = e->dst_input();
     g->RemoveEdge(e);
 
-    if (new_src_output < input_count - resource_input_count) {
+    if (new_src_output < static_cast<int64>(input_count - resource_input_count)) {
```

Comment:
Better to extract RHS to a separate expression of proper type.

I am not sure there is an issue here though. All 3 variables have type `int`.

Reply:
Don't forget about these ones.

Reply:
updated.